projects/cashmere/src/lib/pagination/base-pagination.ts

```diff
@@ -52,6 +52,7 @@ export class BasePaginationComponent extends Initializable implements OnInit {
     set pageSize(value: number) {
         this._pageSize = coerceNumberProperty(value);
         this._pageSizeUpdated();
+        this._emitPageEvent(this.pageNumber);
```

Comment:
🤦‍♂ I can't believe it was that simple!

Reply:
@andrew-frueh looks like you broke some tests though... might want to look into that